pages/common/pio-team.md

```diff
@@ -0,0 +1,32 @@
+# pio team
+
+> Manage PlatformIO teams.
+> More information: <https://docs.platformio.org/en/latest/core/userguide/team/>.
+
+- Create a new team with a description:
+
+`pio team create --description {{description}} {{orgname}}:{{teamname}}`
+
+- Delete a team:
+
+`pio team destroy {{orgname}}:{{teamname}}`
+
+- Add a new member to a team:
+
+`pio team add {{orgname}}:{{teamname}} {{username}}`
+
+- Remove a member from a team:
+
+`pio team remove {{orgname}}:{{teamname}} {{username}}`
+
+- List all teams and their members, the user is part of:
```

Comment:
```suggestion
- List all teams and all their members:
```

Reply:
Don't you think `all teams` could be confused with "all teams there are"?

Reply:
I intended it to mean that—is it not true? https://docs.platformio.org/en/latest/core/userguide/team/cmd_list.html seems to me to mean that.

Reply:
Like in `pio org list`, when I run `pio team list`, I get `You do not have any teams`.

Reply:
What about this?
```suggestion
- List all teams that the user is part of and their members:
```

This seems to me like a word-ordering issue